### PR TITLE
[web] [image] Fixed not accepting props in expo-image like data props

### PR DESCRIPTION
**Pull request by makkarMeenu**
# Why

I couldn't add extra props like data-style in expo-image web.

# How

I have spread all the extra props that user passes which were earlier not there.

# Test Plan

1. Pass the `data-style` to image example in bare-expo application. 
2. Inspect the image on web.
3. See if those props have been added to the rendered image.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-image/CHANGELOG.md [hunk 1]
```diff
@@ -162,3 +162,5 @@ _This version does not introduce any user-facing changes._
 ### 🎉 New features
 
 - Initial release 🥳
+
+- Added all the extra props to expo-image on web ([#22340](https://github.com/expo/expo/pull/22340) by [@makkarMeenu](https://github.com/makkarMeenu))
```
Comment (aleqsio) [inline packages/expo-image/CHANGELOG.md:166]: please move to the unpublished section on the top of the changelog and reword so it's clear that it does prop spread :) 
`Use prop spreading on web to pass all unused props to the native image component`

**Review by aleqsio — CHANGES_REQUESTED** [review]
Thank you for this PR!
This seems to make sense, requested some minor changes to the changelog

File: packages/expo-image/CHANGELOG.md (hunk 1 above)
Comment (makkarMeenu) [reply packages/expo-image/CHANGELOG.md]: Please find the changes in [this](https://github.com/expo/expo/pull/22340/commits/687ab7b5aca364d1f701ed81176764944ff27a8a) commit

**Review by aleqsio — APPROVED** [review]

**Comment by makkarMeenu** [comment]
Hi @aleqsio, some checks are unsuccessful related to sdk/packages and those are not directly related to expo-image. How I am supposed to go about that?

**Comment by aleqsio** [comment]
These seem unrelated, so I've merged the PR :)

**Pull request merged**